### ISSUE TITLE
allowed-hosts failing on urls with port and no scheme

### ISSUE THREAD

**Issue by matthias-ccri** [lirantal/lockfile-lint#93]
Hi there! We've been using lockfile-lint at our company. We have been specifying `--allowed-hosts` with a domain and port, like this: `nexus.example.com:8089`. Up until yesterday, this worked fine:

```
npx lockfile-lint --path package-lock.json --allowed-hosts nexus.example.com:8089 registry.npmjs.org
```

As of today, the above command results in false positives that look like this:
```
detected invalid host(s) for package: zero-crossings@1.0.1-72cbc63a942ee565369f9524fb69fdbe2d885364
    expected: ,registry.npmjs.org
    actual: nexus.example.com:8089
```
(Note the leading comma in the "expected" value. It seems that this CLI argument is being parsed incorrectly?)

Changing `nexus.example.com:8089` to `https://nexus.example.com:8089` makes it work again.

## Expected Behavior

Expected allowed-hosts not to require a scheme. 

## Current Behavior

When using a port in an allowed-host, it doesn't work unless you use a scheme as well. And if that's a feature not a bug, then the error message could be improved.

## Your Environment

- Library Version used: 4.3.3
- Node.js version (e.g. Node.js 5.4): 12.9.1, npm 6.12.1
- Operating System and version (desktop or mobile): linux

**Comment by lirantal** [comment]
FYI you can debug from the CLI via:
```
DEBUG=lockfile* lockfile-lint --path yarn.lock ...
```
It'll show you what has been parsed and a bit more details.

**Comment by lirantal** [comment]
Looked into it quickly, and believe I see the issue.
Thanks for raising this. I'll push a fix and happy if you can confirm it too.

**Comment by lirantal** [comment]
@matthias-ccri should be fixed by PR #94 and a version will be released soon.
It would be greatly appreciated if you could use the latest version and let me know if the issue is fixed.

**Comment by lirantal** [comment]
Fixed in the latest version.
Thanks for reporting @matthias-ccri ❤️

**Comment by matthias-ccri** [comment]
Thanks! Sorry, I wasn't able to figure out how to npx from a git repo. Thanks again for the fix.

**Comment by lirantal** [comment]
Many many thanks to you @matthias-ccri for taking the time to report it. Appreciate it, and happy to push that fix ❤️
